.github/workflows/redeliver.yaml

```diff
@@ -0,0 +1,25 @@
+---
+name: Redeliver failed webhooks
+
+# This workflow runs every 6 hours or when manually triggered.
+on:
+  schedule:
+    - cron: '20 */6 * * *'
+  workflow_dispatch:
+
+# This workflow will use the built in `GITHUB_TOKEN` to check out the repository contents. This grants `GITHUB_TOKEN` permission to do that.
+permissions:
+  contents: read
+
+jobs:
+  redeliver:
+  - name: Redeliver failed webhooks
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v4
+    - uses: gateixeira/retrigger-webhook-action@main
```

Comment:
This action doesn't seem to be an official github action.
Also I think we should never use the `main` version of a third party action we don't control.
And also the usage
https://github.com/gateixeira/retrigger-webhook-action
seems to be totally different. It should not be a scheduled workflow on its own. It says "Add the following snippet to an existing workflow file".
And I think this should really be tested somehow, maybe you can add a dummy workflow in your fork.

Reply:
The alternative is #5876 where we implement it from scratch.

I agree on adding a version, though.

Reply:
> What existing workflow would this be if not this one? I read **existing** as any workflow where this action is useful, and the schedule is what GitHub docs recommend.

I don't know. I don't find the documentation of that action very useful and also not very reassuring :)

Reply:
I never said that I don't like the idea.

And for a proof of concept we still need the proof :)